terraform/envs/gcp/gcp.tfvars

```diff
@@ -13,3 +13,4 @@ oidc_email_domain  = "aisingapore.org\\,aiap.sg"
 oidc_client_id     = "OIDC_CLIENT_ID"
 oidc_client_secret = "OIDC_CLIENT_SECRET"
 
+enable_neo4j = false
```

Comment:
I think `neo4j = true|false` would suffice

Reply:
Modified in [b36784c](https://github.com/aisingapore/kapitan-hull-admin/pull/15/commits/b36784c48f00b5795b10f1a87182c2ef9a31fe78)